sharepoint/sharepoint-ps/sharepoint-online/sharepoint-online.md

```diff
@@ -103,6 +103,10 @@ The following cmdlet references are for SharePoint Online.
 
 {{Manually Enter Get-SPOAppInfo Description Here}}
 
+### [Get-SPOAppPrioritizationPolicies](Get-SPOAppPrioritizationPolicies.md)
+
+{{Gets all existing SPO app prioritization policies of your tenancy}}
```

Comment:
>of your tenancy [](http://example.com/codeflow?start=52&length=15)

```suggestion
in the tenant.
```

Reply:
@pvrk please fix. be sure to add `.` at the end as well.

Reply:
Not our command, Sam. Modifying any text here or nearby is leading to a merge conflict error.